packages/client/src/Conversation.test.tsx

```diff
@@ -41,7 +41,37 @@ it("collapses read messages in a conversation", () => {
       mocks: [$.getConversationMock, $.setIsReadMock]
     }
   )
-  updates(app)
+  await updates(app)
 
-  expect(app.find(CardContent)).toBeTruthy()
+  expect(app.find(Collapse)).toHaveProp("in", true)
+
+  const newConvo = {
```

Comment:
Since you are setting up a new set of test data and a new instance of `Conversation` I think this should go into a new test.